src/Symfony/Component/Yaml/Parser.php

```diff
@@ -381,6 +381,11 @@ private function doParse(string $value, int $flags)
                         if (0 === $this->offset && !$deprecatedUsage && isset($line[0]) && ' ' === $line[0]) {
                             throw new ParseException('Unable to parse.', $this->getRealCurrentLineNb() + 1, $this->currentLine, $this->filename);
                         }
+
+                        if ($this->offset >= 1 && preg_match('/:/', $this->currentLine) && $this->isNextLineIndented()) {
+                            throw new ParseException('Indented blocks after definition are not allowed', $this->currentLineNb + 1, $this->currentLine, $this->filename);
```

Comment:
this needs to go through a deprecation process instead, as done for our previous changes making the parser stricter regarding the spec compliance.

Reply:
@stof Do you have an example I can follow?

Reply:
you can take a look at #23288 for example